pkg/controller/sparkapplication/controller.go

```diff
@@ -1001,3 +1009,17 @@ func (c *Controller) hasApplicationExpired(app *v1beta2.SparkApplication) bool {
 
 	return false
 }
+
+// Clean up resources such as batch scheduler config if applicable.
```

Comment:
Comment needs to be updated.

Reply:
updated it
`Clean up when the spark application is terminated.`